compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/InputDispatcher.skikoMain.kt

```diff
@@ -57,28 +59,37 @@ internal class SkikoInputDispatcher(
     private var modifiers = 0
 
     override fun PartialGesture.enqueueDown(pointerId: Int) {
-        val position = lastPositions[pointerId]!!
         val timeMillis = currentTime
+        val pointers = lastPositions.map {
+            ComposeScene.Pointer(
+                id = PointerId(it.key.toLong()),
+                position = it.value,
+                pressed = true,
+                type = PointerType.Touch
+            )
+        }
```

Comment:
Why do we need to send all the pointers?

Reply:
It was a bug in `ui-test` library - it uses mouse overload to test touches.

> Why do we need to send all the pointers?

Test library must reflect real usage

Reply:
What I meant to ask was - in real usage, do all down pointers get reported each event, even if their state hasn't changed?

Reply:
Yes. And detection of multitouch gestures actively relies on it (zoom, centroid, etc)